docs/workflow/requirements/windows-requirements.md

```diff
@@ -108,3 +108,33 @@ You can also temporarily add a directory to the PATH environment variable with t
 You can make your change to the PATH variable persistent by going to _Control Panel -> System And Security -> System -> Advanced system settings -> Environment Variables_, and select the `Path` variable under `System Variables` (if you want to change it for all users) or `User Variables` (if you only want to change it for the current user).
 
 Simply edit the PATH variable's value and add the directory (with a semicolon separator).
+
+### Windows on Arm64
+
+The Windows on Arm64 development experience has improved substantially over the last few years, however there are still a few steps you should take to improve performance when developing runtime libraries on an ARM device.
+
+Runtime libraries source their compilers from the [Microsoft.NET.Compilers.Toolset](https://www.nuget.org/packages/Microsoft.Net.Compilers.Toolset/) package which currently doesn't bundle native Arm64 bits. This can result in [suboptimal performance](https://github.com/dotnet/runtime/issues/104548) when working on libraries from Visual Studio. The issue can be worked around by [configuring the registry](https://github.com/dotnet/runtime/issues/104548#issuecomment-2214581797) to run the compiler bits as Arm64 processes.
```

Comment:
This is only true right now but won't be true when we approach RC, GA and servicing. Starting with the RC SDK, we will stop using the compilers toolset package (stabilizing the repository).

Also note that the toolset package soon supports arm64 via the app manifest update that is in PR in roslyn. I would prefer not documenting the Registry change as that won't be necessary anymore then (with Windows 24H2 and upwards). But I don't feel strongly about this.

Reply:
The PR seems blocked pending validation on older OS'es. Given that there's going to be an influx of people using ARM devices I'd be inclined that we just merge it and I'll make a note in the tracking issue to remove this as soon as it gets resolved.

Reply:
That's fine to me but the toolset package statement is still in-accurate. I would phrase this differently, i.e. "during preview releases ..."

Reply:
Ok, I've prefixed the paragraph with that statement.

Reply:
> Starting with the RC SDK, we will stop using the compilers toolset package (stabilizing the repository).

@ViktorHofer Is this going to be new for .NET 9?

It is not what we have done for .NET 8. Roslyn gets regularly updated via subscription. We have not stopped with RCs. For example, the release/8.0 branch is building with `4.8.0-7.23566.2` compilers currently that are not the ones in the SDK.

Reply:
(I think that how we have done it for .NET 8 is fine.)